client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -224,5 +227,11 @@ void main(void)
 	col = mix(skyBgColor, col, clarity);
 	col = vec4(col.rgb, base.a);
 
+#ifdef SECONDSTAGE
+	gl_FragData[0] = col;
+	gl_FragData[1] = vec4(normalPass, DRAW_TYPE);
+	gl_FragData[2] = vec4((fogDistance-length(eyeVec))/1000, 0,0,0);
```

Comment:
Why not simply `length(eyeVec)`? Also do you need to scale it if the render target format is floating-point?